store/extensions/Q-Wired-Page/extension.json

```
@@ -0,0 +1,40 @@
+{
+  "title": "Q-Wired-Page",
+  "description": "Removes the parent variable menu item and adds the children to the Wired menu.",
+  "authors": [
+    {
+      "name": "MrGeneralQ",
+      "hotel": ".nl",
+      "username": "MrGeneralQ",
+      "discord": "MrGeneralQ",
+      "twitter": ""
+    }
+  ],
```

```suggestion
  "authors": [
    {
      "name": "MrGeneralQ",
      "hotel": ".nl",
      "username": "MrGeneralQ",
      "discord": "mrgeneralq"
    }
  ],
```
You can leave out the empty ones and discord usernames are lowercase